src/babel.js

```diff
@@ -44,11 +44,7 @@ exports.getCachePath = function (sourceCode) {
 exports.compile = function (sourceCode, filePath) {
   if (!babel) {
     babel = require('babel-core')
-    var Logger = require('babel-core/lib/transformation/file/logger')
-    var noop = function () {}
-    Logger.prototype.debug = noop
-    Logger.prototype.verbose = noop
-
+    require('debug').log = () => { } // Stop Babel logging to stdout/stderr (would error when no console attached)
```

Comment:
Yes, it will. But if you use debug.log on Windows right now when launching from the ui with that process variable set it will error.

Short of getting the debug people to not try and write to stderr/stdout on Windows I don't know there is a great solution.